src/guides/v2.3/release-notes/release-notes-2-3-5-commerce.md

```diff
@@ -1191,6 +1193,21 @@ We have fixed hundreds of issues in the Magento 2.3.5 core code.
 
 *  The WYSIWYG editor no longer hangs indefinitely when you try to upload an image from the Admin. Previously, the image upload popup window hung until you refreshed the page. *Fix submitted by Nazar Klovanych in pull request [25556](https://github.com/magento/magento2/pull/25556)*. [GitHub-23966](https://github.com/magento/magento2/issues/23966)
 
+## Known issues
+
+**Issue**: Magento prompts customers to log in by displaying this message: `This account is not confirmed. <a href="Base URLcustomer/account/confirmation/?email=customer70%40yopmail.com">Click here</a> to resend confirmation email`.  The **Click here** link in this message should open the Send confirmation link page, but is currently inactive. Magento instead displays this message: `This account is not confirmed`. Click here to resend confirmation email`. **Workaround**: A hot fix will soon be available for this issue, and a fix will be available in Magento 2.3.6, with is scheduled for release in Q4 2020. <!--- MC-33148-->
+
+**Issue**: The Compare Products feature does not work as expected in deployments with multiple store views. When a user tries to compare products from multiple store views and one product has an empty value for a comparable attribute, Magento displays a corrupted Compare Products page. **Workaround**: Comparable attribute values cannot be empty. Merchants should specify non-empty values for comparable product attributes or use the default storeview value for the attribute. A fix will be available in Magento 2.3.6, which is scheduled for release in Q4 2020. <!--- MC-33203-->
+
+**Issue**: Magento throws an error on the Order Summary  section of the checkout workflow when a customer removes store credit after first selecting it as a payment method for the entire order. **Workaround**: Customers can refresh the Order page by pressing **F5**. A fix will be available in Magento 2.3.6, which is scheduled for release in Q4 2020.
```

Comment:
"by pressing F5" is a very specific shortcut which doesn't work everywhere. For example it doesn't work on macOS when using Chrome or Safari (it works in Firefox though).
I would just say "by refreshing the page", since that's a more general instruction 🙂

Reply:
changed -- thanks for catching.

Reply:
changed in source. Thanks!